src/main/java/com/googleinterns/zoomtube/data/Lecture.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.auto.value.AutoValue;
+
+/** Stores data for lectures added to site. */
+@AutoValue
+public abstract class Lecture {
+  /** Used to create an entity and its fields. */
+  public static final String ENTITY_KIND = "Lecture";
+  public static final String PROP_NAME = "lectureName";
+  public static final String PROP_URL = "videoUrl";
+  public static final String PROP_ID = "id";
+  public static final String PROP_VIDEO_ID = "video-id";
```

Comment:
Do these need to be public? If not please make private. Always go with making things private and increase visibility only as needed.

Reply:
yes they need to be public because `LectureServlet.java` references them

Reply:
Please move these to a separate class so Lecture and LectureServlet can reference them.

Reply:
moved to `LectureEntityFields.java`